CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changelog and versioning
 0.0.23
 ------
 
-- Fix bug with legacy queries sending nested functions in a different structure
```

Comment:
Why did you delete this line ?

Reply:
That was the PR I closed because I fixed it on Sentry's side. https://github.com/getsentry/snuba-sdk/pull/73